Jenkinsfile

```diff
@@ -35,7 +35,7 @@ pipeline {
 				branch 'master'
 			}
 			steps {
-				build wait: false, job: '/tycho-sitedocs'
+				sh 'mvn -V clean install site site:stage'
```

Comment:
I think it would be nice speed up to not execute the tests aka -DskipTests=true. It's done in the sitedocs job too.

Reply:
I should have expand the job config :-D